es5-shim.js

```diff
@@ -176,6 +179,36 @@ var ES = {
     }
 };
 
+/**
+ * An implementation of `slice`. Primarily for use with `arguments`.
```

Comment:
Can't find `sliceArgs`? Need it instead of `Array#slice` because I believe that in some of the environments that these fixes are for, it doesn't slice arguments.